package.json

```diff
@@ -1,7 +1,7 @@
 {
   "name": "react-polymorph",
   "description": "React components with highly customizable logic, markup and styles.",
-  "version": "0.5.5",
+  "version": "0.5.6",
```

Comment:
The version here should not be changed in PRs but on `master` right before we publish and tag the next release 😉  please revert this change